sdk/eventhub/Microsoft.Azure.EventHubs/Directory.Build.props

```diff
@@ -1,5 +1,6 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <ExcludeFromCodeCoverage>true</ExcludeFromCodeCoverage>
```

Comment:
I'm not a fan of starting to exclude projects one by one. EH is not the only directory that has T1&T2 clients so we need a general solution here before we jump into workarounds.

Reply:
That only affects T1. Somehow - no way I could find after going through their various targets, nor does @jsquire know - they are detected as T2. None of the other CIs that ran as part of this PR exhibited that.

Reply:
Note from the targets file I added only T2 projects are covered by default. These are an anomaly.